packages/manager/modules/cloud-connect/src/details/statistics/translations/Messages_fr_FR.json

```
@@ -0,0 +1,12 @@
+{
+  "cloud_connect_stats_title": "Statistiques",
+  "cloud_connect_stats_type": "Type",
+  "cloud_connect_stats_type_light": "Lumière",
+  "cloud_connect_stats_type_traffic": "Traffic",
```

Not sure about this suggestion, please let me know:
```suggestion
  "cloud_connect_stats_type_traffic": "Trafic",
```